src/test/java/dev/jbang/util/TestCommandBuffer.java

```diff
@@ -45,16 +45,18 @@ void testRunWinPS1() {
 	}
 
 	@Test
+	@EnabledOnOs(OS.WINDOWS)
 	void testApplyWindowsMaxLengthLimitExe() throws IOException {
 		ProcessBuilder pb = CommandBuffer.of(argsTooLong("foo.exe"))
 			.shell(Util.Shell.cmd)
 			.applyWindowsMaxLengthLimit()
 			.asProcessBuilder();
-		assertThat(pb.command().size(), greaterThan(2));
-		assertThat(pb.command().get(1), not(anyOf(startsWith("@"), startsWith("\"@"))));
+		assertThat(pb.command().size(), equalTo(2));
```

Comment:
This is wrong. The test is exactly meant to show that it is not necessary to use the @argFiles for executables.